### PR TITLE
Add 'Added By' to Contact Details

### PR DESCRIPTION
**Pull request by andvirga**
Jira: https://bugs.benetech.org/browse/CHI-612

Related to: https://github.com/tech-matters/flex-plugins/pull/424

Primary reviewer: @murilovmachado

### CODE REVIEW

**Comment by nickhurlburt** [comment]
Keep in mind that this will need to work for clients that aren't using 'Added by' yet.  We may deploy HRM for all production, but only update one client (eg, South Africa).  It should still work for Zambia.  I think this will do that, just a reminder for both code author and reviewer to check for that.

File: controllers/contact-controller.js [hunk 1]
```diff
@@ -201,6 +201,7 @@ function convertContactsToSearchResults(contacts) {
       const { callType, caseInformation } = contact.rawJson;
       const categories = retrieveCategories(caseInformation.categories);
       const counselor = contact.twilioWorkerId;
+      const addedBy = contact.createdBy;
```
Comment (GPaoloni) [inline controllers/contact-controller.js:204]: The only thought I have is that we should send the raw data from DB, and then we can rename as we want in the fronted, so I prefer this to be called `createdBy`. Up to you and the team tho, just sharing a thought.

**Review by GPaoloni — APPROVED** [review]

File: controllers/contact-controller.js (hunk 1 above)
Comment (murilovmachado) [reply controllers/contact-controller.js:204]: I agree with gian about the naming.

**Review by murilovmachado — APPROVED** [review]
Code and behavior look good to me!

**Pull request merged**